src/NuGet.Core/NuGet.Packaging/Signing/Archive/SignedPackageArchiveIOUtility.cs

```diff
@@ -709,13 +746,15 @@ private static long WriteFileIntoZip(BinaryWriter writer, byte[] fileData)
         /// Writes the end of central directory header into a zip using the writer starting at the writer.BaseStream.Position.
         /// The new end of central directory record will be based on the one at reader.BaseStream.Position.
         /// </summary>
-        /// <param name="reader">BinaryWriter to be used to read exisitng end of central directory record.</param>
-        /// <param name="writer">BinaryWriter to be used to write file.</param>
+        /// <param name="reader">BinaryReader to be used to read the existing end of central directory record.</param>
+        /// <param name="writer">BinaryWriter to be used to write the updated end of central directory record.</param>
+        /// <param name="entryCountChange">The change to central directory header counts.</param>
         /// <param name="sizeOfSignatureCentralDirectoryRecord">Size of the central directory header for the signature file.</param>
         /// <param name="sizeOfSignatureFileHeaderAndData">Size of the signature file and the corresponding local file header.</param>
         private static void ReadAndWriteUpdatedEndOfCentralDirectoryRecordIntoZip(
             BinaryReader reader,
             BinaryWriter writer,
+            sbyte entryCountChange,
```

Comment:
why use `sbyte` data type instead of `int`?

Reply:
`entryCountChange` must be either -1 or +1.  `sbyte` is the smallest integer type that supports these values.  Choice of type is fairly insignificant since this method is private anyway.